### PR TITLE
Added publications and media per Melanie

### PR DESCRIPTION
**Pull request by johncarl81**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -229,30 +237,102 @@ <h2>Valles Caldera</h2>
...
+                            platforms: a mobile uncrewed ground vehicle (UGV) equipped with a LiDAR sensor, an uncrewed aerial vehicle (UAV)
+                            with a gimbal-mounted stereo camera for air-to-air inspections, and a UAV with a capture mechanism equipped with radars and camera...</p>
+
+                        <h3><a href="https://ieeexplore.ieee.org/abstract/document/10025390">Robust Linear-Velocity-Free Formation Tracking of Multiple Quadrotors with Unknown Disturbances</a></h3>
+                        <i>J. Lin, Y. Wang, Z. Miao, Q. Lin, G. Hu, and R. Fierro</i>
+                        <p>This article addresses the problem of quadrotor formation control with inaccessible linear
+                            velocity and the unknown disturbances under directed interaction topologies. Considering the
+                            unknown disturbances act in both the translational and rotational motions of each underactuated
+                            quadrotor, a hierarchical distributed disturbance rejection controller is developed for the
+                            quadrotor formation...</p>
```
Comment (Carter90) [inline index.html]: @johncarl81 What is the purpose of the `...`s here? Is that just where the abstracts got cut off?

**Comment by Carter90** [comment]
@johncarl81 It might be worth it to pop the Publications section now we have a bunch of content, one div level up so that the formatting looks better on full size sites, so we don't have an empty column on the right for half the page
[Screencast from 09-19-2023 06:11:58 PM.webm](https://github.com/BCLab-UNM/volcan_website/assets/27081199/7d087668-8c8b-498c-bbeb-284ef113d131)

File: index.html (hunk 1 above)
Comment (johncarl81) [reply index.html]: The abstracts are huge.  I abbreviated them.

Comment (Carter90) [reply index.html]: cool so do we want to keep the `...`s?

Comment (Carter90) [reply index.html]: Sorry If you did not get the notification but @johncarl81 do we want to keep the `...`s? if so then looks good and I'll merge and do a pull on the cs systems

Comment (johncarl81) [reply index.html]: Yeah, lets keep them.  We actually had them [previously](https://github.com/BCLab-UNM/volcan_website/blob/master/index.html#L239).

**Review by Carter90 — APPROVED** [review]
Cool looks good

**Pull request merged**